server/adaptors/integrations/integrations_builder.ts

```diff
@@ -25,6 +25,7 @@ export class IntegrationInstanceBuilder {
     template: IntegrationTemplate,
     options: BuilderOptions
   ): Promise<IntegrationInstance> {
+    console.log(template);
```

Comment:
Is there a standard for logging or is just logging templates fine?

Reply:
removed here: [02375c4](https://github.com/Swiddis/dashboards-observability/pull/2/commits/02375c4232ea6174a451aa486a2b62a864dd124a) that was just for testing